config/custom_items.txt

```diff
@@ -629,6 +629,13 @@ item_path: /obj/item/weapon/fluff/dragor_dot
 
 # ######## R CKEYS
 
+{
+ckey: radiantaurora
+character_name: Tiemli Kroto
+item_path: /obj/item/clothing/glasses/welding/tiemgogs
+req_access = 29
```

Comment:
From a past addition to this, we got https://github.com/VOREStation/VOREStation/pull/5649/files#diff-9b4bb1421d29604d1da001449fc58360R325 to look at rather than having `req_access = 29`.